roles/dtc/common/templates/ndfc_route_control.j2

```diff
@@ -0,0 +1,26 @@
+{% for policy_types in group_item %}
+  {% if "route_maps" in policy_types %}
+    {% include "ndfc_route_map.j2" %}
```

Comment:
@ccoueffe this will apply to all the other includes as well, but can you please group the different include j2 files for this into a folder under ```templates``` and call it ```ndfc_route_control``` and then update the include lines in this file? Thanks!